decompress_cap_128m.c

```diff
@@ -45,12 +45,14 @@
 #endif
 
 static const char* otype_suffix(uint32_t otype) {
+    // clang-format off
     switch (otype) {
 #define OTYPE_CASE(Name, ...)                                                                                          \
     case CC128M_##Name: return " (CC128M_" #Name ")";
-    CC128M_LS_SPECIAL_OTYPES(OTYPE_CASE, )
+        CC128M_LS_SPECIAL_OTYPES(OTYPE_CASE, )
```

Comment:
Deindent

Reply:
Not sure how that ended up in the diff. Fixed.